src/redux/actions/cellSets/loadCellSets.js

```diff
@@ -7,19 +7,22 @@ import endUserMessages from 'utils/endUserMessages';
 
 const loadCellSets = (experimentId, forceReload = false) => async (dispatch, getState) => {
   const {
-    loading, error, updatingClustering,
+    loading, error, updatingClustering, initialLoadPending,
   } = getState().cellSets;
 
-  if (!forceReload && ((!loading && !error) || updatingClustering)) {
-    return null;
-  }
+  const loadingAlreadyReserved = loading || updatingClustering;
```

Comment:
what does `loadingAlreadyReserverd` mean? the name is not very clear to me

Reply:
It means that either:
> load request has already been sent
or the clustering is being updated, so there's no point in fetching outdated cell sets until the new cell sets clustering is done. 

I'll look into changing the name

Reply:
Renamed to `loadingBlocked`